plugins/modules/zos_copy.py

```diff
@@ -2245,6 +2245,37 @@ def normalize_line_endings(src, encoding=None):
     return src
 
 
+def source_locked(dataset_name):
```

Comment:
This is not a blocker but only a comment also to the reviewers @fernandofloresg ,@richp405 , @AndreMarcel99 this function would have been a good candidate for a module_utils entry and not solely accessible to zos_copy. Other modules such as zos_blockinfile, zos_lineinfile , zos_apf all write to data sets (even zos_data_set when it creates members) thus we could use this in a more general and broader sense. Something to note when reviewing. Ill open an issue to move it since we are up against a release point.